src/_P109_ThermOLED.ino

```diff
@@ -41,7 +41,7 @@
    -  /control?cmd=thermo,mode,0               Switch heating off, absolutely do nothing until further notice
 
    ------------------------------------------------------------------------------------------
-   Copyleft Nagy Sándor 2018 - https://bitekmindenhol.blog.hu/
+   Copyleft Nagy Sandor 2018 - https://bitekmindenhol.blog.hu/
```

Comment:
Don't think that should have been changed...

Reply:
Me too, apparently it happened on the clipboard...